__tests__/utils.test.ts

```diff
@@ -0,0 +1,88 @@
+// ./__tests__/utils.test.ts
+//
+// Unittests for the L-Dist utility functions.
+
+// Vitest essentail imports.
+import { describe, expect, it } from 'vitest';
+
+// Local utility functions import.
+import {
+  calculateScore,
+  getMatchingIndices,
+  getMaxLevenshteinDistance,
+  levenshteinFullMatrixSearch,
+} from '../src/utils';
+
+// levenshteinFullMatrixSearch test suite.
+describe('levenshteinFullMatrixSearch', () => {
+  it('returns a matrix of the correct dimensions', () => {
+    const query = 'hello';
+    const target = 'world';
+    const matrix = levenshteinFullMatrixSearch(query, target);
+    expect(matrix.length).toBe(query.length + 1);
+    expect(matrix[0].length).toBe(target.length + 1);
+  });
+
+  it('returns a matrix with the correct values', () => {
+    const query = 'hello';
+    const target = 'world';
+    const matrix = levenshteinFullMatrixSearch(query, target);
+    expect(matrix[0][0]).toBe(0);
+    expect(matrix[0][1]).toBe(1);
+    expect(matrix[1][0]).toBe(1);
+    expect(matrix[1][1]).toBe(1);
+    expect(matrix[1][2]).toBe(2);
+    expect(matrix[5][5]).toBe(4);
+  });
+});
+
+// getMaxLevenshteinDistance test suite.
+describe('getMaxLevenshteinDistance', () => {
+  it('returns strictly 3 for short len 0~5 strings', () => {
```

Comment:
```suggestion
  it('should return strictly 3 for short len 0~5 strings', () => {
```
can we add `should` before every test ? That sets a better context on what the test is supposed to do  "should return x when y happens" instead of "returns x when y happens"

Reply:
@ad1992

- [x] Refactored all `verbs` into `should verb` in 9bfc05c.